atlp-rwanda/vikings-ec-bn
# #184347018 Implementing email verification

#### What does this PR do?
This PR implements email verification

#### Description of Task to be completed?
- Send email verification to the user
- Handle user request with token to confirm email

#### How should this be manually tested?
- Clone the [repo](https://github.com/atlp-rwanda/vikings-ec-bn)
- Checkout branch `feature/verify-email-184347018`
- Run command  **npm install**
- Add **EMAIL_USERNAME** variable with value **vikingsatlp@gmail.com** in **.env** 
- Add **EMAIL_PASSWORD** variable in **.env** with shared password
- Add **VERIFY_URL** variable with value **http://localhost:5000/api/v1/users/verify-email** in **.env**
- Open http://localhost:5000/api-docs in your browser 
- Try register endpoint
- Check your email
- Click the link in received email or use **api/v1/users/verify-email/{token}** from api/docs

#### What are the relevant pivotal tracker stories?
[#184347018](https://www.pivotaltracker.com/story/show/184347018)

#### Screenshots (if appropriate)

![email](https://user-images.githubusercontent.com/90104592/219014572-4cb65894-791e-4ca6-b294-78fcbc2f394d.PNG)
